docs/guide/dev/integration/vue/README.md

```diff
@@ -329,6 +329,32 @@ Corresponds to the {@linkapi CKEDITOR.editor#event-change `change`} editor event
 <ckeditor @input="onEditorInput"></ckeditor>
 ```
 
+### `namespaceloaded`
+
+Fired once {@linkapi CKEDITOR} namespace is loaded on the page. This event is called only one time for every editor instance loaded on the page and only when using [`editor-url`](#editor-url) directive with valid CKEditor 4 CDN URL. Allows to modify global editor namespace.
+
+```html
+<ckeditor @namespaceloaded="onNamespaceLoaded" :config="editorConfig"></ckeditor>
```

Comment:
To be consistent with the previous snippets (like [`editor-url`](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_vue.html#editor-url)) we should wrap `ckeditor` component into `<template>` and `<div id="app">` tags.